site/guide/register-models-in-model-inventory.qmd

```diff
@@ -26,7 +26,8 @@ To be able to document models using ValidMind's model documentation and validati
    - Select the relevant model methodology being used
    - Select the relevant model use case
    - Provide a purpose statement to explain what the model will be used for
-   - select the preliminary risk tier for the model
+   - Select the preliminary risk tier for the model
+   - If the model is a [vendor model](https://docs.validmind.ai/about/glossary/glossary.html#vendor-model), toggle **Is Vendor Model** and provide the vendor name.
```

Comment:
```suggestion
   - If the model is a [vendor model](../about/glossary/glossary.qmd#vendor-model), toggle **Is Vendor Model** and provide the vendor name.
```

Use a relative link. This works, I tested it, though we do have an issue with the navbar at the top covering the text.